bridges/MspabooruBridge.php

```diff
@@ -1,12 +0,0 @@
-<?php
-require_once('GelbooruBridge.php');
-
-class MspabooruBridge extends GelbooruBridge {
-
-	const MAINTAINER = 'mitsukarenai';
-	const NAME = 'Mspabooru';
-	const URI = 'http://mspabooru.com/';
-	const DESCRIPTION = 'Returns images from given page';
-	const PIDBYPAGE = 50;
-
```

Comment:
Why we delete this bridge and some others?

Reply:
Ah this is a mistake on my part. I though this was a dead and empty skeleton bridge. I can see now that it inherits from another bridge.